packages/react-scripts/template/README.md

```diff
@@ -1972,6 +1980,44 @@ npm run build
 npm run analyze
 ```
 
+### Using Webpack Stats JSON
```

Comment:
Should we add a note that this feature is available only in some specific version of `react-scripts`, like many other parts of the README have?

> Note: this feature is available with `react-scripts@x.y.z` and higher.

I don't know how the process of adding these version notes goes, though. Are they added just before releasing a new version?

Reply:
Yeah, good idea!

I'm also unfamiliar with the process, since yeah I don't know which version it'll be released in. Happy to update the PR if this is knowable in advance, though :)

Reply:
Please remove this part or hide it behind comments. 2.x is the main branch now so that's what you find when you look at the repository. It will be deeply confusing to the users to see this in README now.

Reply:
https://github.com/facebook/create-react-app/pull/4564